classes/admin/class-ledyer-meta-box.php

```diff
@@ -69,6 +69,39 @@ public function meta_box_content() {
 		}
 	}
 
+	/**
+	 * Get test environment display name
+	 * @return string
+	 */
+	protected function get_test_environment_name() {
+		switch (ledyer()->get_setting( 'development_test_environment' )) {
+			case 'local':
+				return "Local";
+			case 'development':
+			case 'local-fe':
+				return "Development";
+			default: 
+				return "Sandbox";
+		}
+	}
+
+
+	/**
+	 * Get ledyer merchant portal test url
+	 * @return string
+	 */
+	protected function get_ledyer_merchant_portal_test_url() {
+		switch (ledyer()->get_setting( 'development_test_environment' )) {
+			case 'local':
+				return 'http://host.docker.internal:2000/orders/';
```

Comment:
Isn't this the accurate value to use here?
`'http://localhost:2000/orders/'`

Reply:
Not sure to be honest, I don't have the full picture really :) Depends on if Ledyer merchant portal is dockerized or not locally I guess. The above was @plarsson suggestion when I asked him about it, but I might have been to vague in describing what I needed perhaps :D

Reply:
OK, let's keep it then:)